src/pages/new/index.js

```diff
@@ -131,6 +133,7 @@ export default () => {
         }}
         playButtonLink="/foo"
       />
+      <Spacer heightDesktop="42px" heightMobile="0" />
```

Comment:
This just feels like it should be part of the Video Spotlight component itself, since this appear to be 100% overlapping with the bottom section of the video background image.

Reply:
Ill make a ticket for the subtle correction

Reply:
https://app.clubhouse.io/sheltertech/story/1747/fix-padding-and-background-for-the-video-spotlight-component